specification/cloudhealth/CloudHealth.Management/main.tsp

```diff
@@ -0,0 +1,843 @@
+}
+
+@doc("HealthModel properties")
+model HealthModelProperties {
+  @visibility(Lifecycle.Read)
+  @doc("The data plane endpoint for interacting with health data")
+  dataplaneEndpoint?: string;
+
+  @visibility(Lifecycle.Read)
+  @doc("The status of the last operation.")
+  provisioningState?: HealthModelProvisioningState;
+
+  @doc("Configure to automatically discover entities from a given scope, such as a Service Group. The discovered entities will be linked to the root entity of the health model.")
+  discovery?: ModelDiscoverySettings;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "Self-explanatory"
+@doc("Health Model provisioning states")
+union HealthModelProvisioningState {
+  string,
+  ResourceProvisioningState, // include standard provisioning states
+  Creating: "Creating",
+  Deleting: "Deleting",
+}
+
+@doc("Settings for automatically discovering entities for the health model.")
+model ModelDiscoverySettings {
+  @doc("The scope from which entities should be automatically discovered. For example, the resource id of a Service Group.")
+  scope: string4096;
+
+  @doc("Whether to add all recommended signals to the discovered entities.")
+  addRecommendedSignals: DiscoveryRuleRecommendedSignalsBehavior;
+
+  @doc("Which Managed Identity of the health model to use for discovery. Defaults to SystemAssigned, if not set. Can be set to 'SystemAssigned' or to the resource id of a user-assigned managed identity which is linked to the health model.")
+  identity?: string4096;
+}
+
+@doc("A signal definition in a health model")
+@parentResource(HealthModel)
+model SignalDefinition is ProxyResource<SignalDefinitionProperties> {
+  @doc("Name of the signal definition. Must be unique within a health model.")
+  @key("signalDefinitionName")
+  @segment("signaldefinitions")
+  @path
+  name: stringResourceName;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "Self-explanatory"
+@doc("Supported signal kinds as discriminator")
+union SignalKind {
+  string,
+  AzureResourceMetric: "AzureResourceMetric",
+  LogAnalyticsQuery: "LogAnalyticsQuery",
+  PrometheusMetricsQuery: "PrometheusMetricsQuery",
+}
+
+@doc("SignalDefinition properties")
+@discriminator("signalKind")
+model SignalDefinitionProperties {
+  @visibility(Lifecycle.Read)
+  @doc("The status of the last operation.")
+  provisioningState?: HealthModelProvisioningState;
+
+  @doc("Display name")
+  @minLength(1)
+  @maxLength(260)
+  displayName?: string;
+
+  @doc("Kind of the signal definition")
+  signalKind: SignalKind;
+
+  @doc("Interval in which the signal is being evaluated. Defaults to PT1M (1 minute).")
+  refreshInterval?: RefreshInterval = "PT1M";
+
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "approved scenario for use as Tags alternative in proxy resource"
+  @doc("Optional set of labels (key-value pairs)") // Since Signal Definition is a proxy resource, we cannot use Azure Tags
```

Comment:
>// Since Signal Definition is a proxy resource, we cannot use Azure Tags [](http://example.com/codeflow?start=51&length=72)

I thought you CAN use tags

Reply:
according to this, (pure) proxies still dont support tags

https://eng.ms/docs/products/arm/rp_onboarding/tracked_vs_proxy_resources